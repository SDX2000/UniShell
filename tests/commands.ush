echo "================"
echo $SCRIPT_NAME
echo "================"

echo "INIT_DIR=$INIT_DIR"
echo "SCRIPT_PATH=$SCRIPT_PATH"
echo "SCRIPT_DIR=$SCRIPT_DIR"
echo "SCRIPT_NAME=$SCRIPT_NAME"