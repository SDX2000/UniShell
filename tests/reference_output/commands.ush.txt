================
commands.ush
================
INIT_DIR=/home/sandeepd/source/tools/unishell/tests
SCRIPT_PATH=/home/sandeepd/source/tools/unishell/tests/commands.ush
SCRIPT_DIR=/home/sandeepd/source/tools/unishell/tests
SCRIPT_NAME=commands.ush
